storage/innobase/include/os0file.h

```diff
@@ -274,8 +274,12 @@ static const char ENCRYPTION_KEY_MAGIC_V2[] = "lCB";
 information version. */
 static const char ENCRYPTION_KEY_MAGIC_V3[] = "lCC";
 
+/** Encryption magic bytes before 8.0.19, it's for checking KEYRING
+redo log information version. */
 static const char ENCRYPTION_KEY_MAGIC_RK_V1[] = "lRK";
 
+/** Encryption magic bytes for 8.0.19+, it's for checking KEYRING
+redo log information version. */
```

Comment:
Also just wanted to know what these letters meant "lRK" and "RKB" ?

Reply:
I guess lRK was for Rotated Keys. The other versions names like PSA was easy to upgrade PSB. Here lRB would be confusing, so I changed the version name to RKB - next will be RKC etc.